app/model/orm/experiment.py

```diff
@@ -59,10 +60,55 @@ class Experiment(OrmBase):
     )
 
     measurementContexts: Mapped[List['MeasurementContext']] = relationship(
+        order_by='MeasurementContext.subjectTypeOrdering, MeasurementContext.subjectName',
         secondary='Bioreplicates',
         viewonly=True,
     )
 
+    def get_df(self, db_session):
```

Comment:
This is a data export for an entire experiment, but quite a lot more columns are needed to show the exact context of each row. I may rework this to show ids instead of names in certain places, but we'll see.